### PR TITLE
Allow customization of the metrics path

### PR DESCRIPTION
**Pull request by omegas27**
Allow configuring promxy's metrics path

### CODE REVIEW

File: cmd/promxy/main.go [hunk 1]
```diff
@@ -74,6 +74,8 @@ type cliOpts struct {
 	WebCORSOriginRegex string        `long:"web.cors.origin" description:"Regex for CORS origin. It is fully anchored." default:".*"`
 	WebReadTimeout     time.Duration `long:"web.read-timeout" description:"Maximum duration before timing out read of the request, and closing idle connections." default:"5m"`
 
+	MetricsPath string `long:"metrics-path" description:"url path" default:"/metrics"`
```
Comment (jacksontj) [inline cmd/promxy/main.go:77]: nit: the description here should specify that this is the path for the prometheus metrics endpoint.

**Review by jacksontj — COMMENTED** [review]
Thanks for picking up this TODO! :) one minor change requested; but otherwise looks great!

File: cmd/promxy/main.go (hunk 1 above)
Comment (omegas27) [reply cmd/promxy/main.go]: My latest change should address that :)

**Review by jacksontj — APPROVED** [review]

**Comment by jacksontj** [comment]
Perfect, Thanks for the contribution!

**Pull request merged**